### ISSUE TITLE
ValueError: Could not determine alphabet for seq_type linear unknown  

### ISSUE THREAD

**Issue by unawareK** [quadram-institute-bioscience/albatradis#15]
Hi, i've been trying to run AlbaTradis using both the example datasets and my own, but this error keeps popping up:

 File "/Users/s2984406/opt/anaconda3/lib/python3.7/site-packages/Bio/GenBank/__init__.py", line 1424, in record_end
    "Could not determine alphabet for seq_type %s" % self._seq_type
ValueError: Could not determine alphabet for seq_type linear unknown

Is there a way to rectify this? Thanks!

**Comment by sbastkowski** [comment]
Hi,
could you please send me the full command you are using (on the example data). 
Cheers

**Comment by unawareK** [comment]
Hi,

The full command is the same one I copied from the page:

albatradis -v -a reference_BW25113.embl 025mgLTricRep1.insert_site_plot.gz 
 25mgLTricRep2.insert_site_plot.gz controlLBrep1.insert_site_plot.gz controlLBrep2.insert_site_plot.gz

Thanks!

**Comment by sbastkowski** [comment]
Did you run, albatradis -h to see if it is installed properly?

**Comment by unawareK** [comment]
Yup, and it looks like its installed properly.

I tried the same command without the "-a" argument, and it appears to have worked.

However, as i understand it, this used a sliding window instead of the reference annotations, and the output was a little hard to make sense of.

**Comment by sbastkowski** [comment]
So I used a clean VM (ubuntu 18.04), run the conda installation: conda install -c conda-forge -c bioconda albatradis
I then cloned the git repo (to get the data), went into the data/albatradis_data directory and run:
albatradis -v -a reference_BW25113.embl 025mgLTricRep1.insert_site_plot.gz 025mgLTricRep2.insert_site_plot.gz controlLBrep1.insert_site_plot.gz controlLBrep2.insert_site_plot.gz

It runs fine. I think there is probably an issue with your installation. Which method did you use?
